pkg/permissions/claims.go

```diff
@@ -88,3 +89,16 @@ func (claims *Claims) Expired() bool {
 	validUntil := claims.IssuedAtUTC().Add(validityDuration)
 	return validUntil.Before(time.Now().UTC())
 }
+
+// CreateCode returns a new sharecode to put the codes field of a permissions document
+func CreateCode(oauthSecret []byte, domain, name string) (string, error) {
```

Comment:
nit: why not using the `instance.MakeJWT()` ?

Reply:
I've just extracted some code to use at another place, but I should use `instance.MakeJWT()`